nvi-rest/src/main/java/no/sikt/nva/nvi/fetch/FetchNviCandidateHandler.java

```diff
@@ -0,0 +1,41 @@
+package no.sikt.nva.nvi.fetch;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.URI;
+import java.util.Optional;
+import no.sikt.nva.nvi.common.service.NviService;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.NotFoundException;
+import nva.commons.core.Environment;
+
+public class FetchNviCandidateHandler extends ApiGatewayHandler<Void, FetchCandidateResponse> {
+
+    public static final String PARAM_CANDIDATE_IDENTIFIER = "candidateIdentifier";
+    private final NviService service;
+
+    public FetchNviCandidateHandler(NviService service, Environment environment) {
+        super(Void.class, environment);
+        this.service = service;
+    }
+
+    @Override
+    protected FetchCandidateResponse processInput(Void input, RequestInfo requestInfo, Context context)
+        throws ApiGatewayException {
+        return attempt(() -> requestInfo.getPathParameter(PARAM_CANDIDATE_IDENTIFIER))
+                   .map(URI::create)
```

Comment:
Is path param candidateIdentifier a URI or UUID? In the tests you are testing with a URI, but in the open api spec it seems like it is an UUID?

Reply:
@brinxmat A penny for your thoughts.
.../nvi/candidate/d5fa52eb-96da-4bc1-add9-979cc0978dfa
.../nvi/candidate/http%3A%2F%2Fnva.sikt.no%2Fpublication%2Fd5fa52eb-96da-4bc1-add9-979cc0978dfa

Reply:
They cost more than a penny :(